MonstieWash/Assets/Scripts/VFX/ToolFX.cs

```diff
@@ -0,0 +1,43 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+
+public class ToolFX : MonoBehaviour
+{
+    [SerializeField] private ParticleSystem particlesOnUse;
+    ParticleSystem myParticles;
```

Comment:
Just convention stuff.
```suggestion
    private ParticleSystem m_myParticles;
```